website/app/styles/pages/about/principles.scss

```diff
@@ -35,9 +35,10 @@
 }
 
 .doc-content-principles__title {
-  @include doc-font-style-h3();
   margin: 0;
   color: var(--doc-color-black);
+
+  @include doc-font-style-h3();
```

Comment:
question: was there a reason for moving this to the end of the class? Asking as we generally tend to include them at the top of the class (there are a few other instances in this file as well)

Reply:
Yes, I did it to resolve this warning: https://sass-lang.com/documentation/breaking-changes/mixed-decls/